src/components/AddBookForm.js

```diff
@@ -1,13 +1,43 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
 import PropTypes from 'prop-types';
+import { addBook } from '../redux/books/books';
 import './AddBookForm.css';
 
 export default function AddBookForm({ categories }) {
```

Comment:
- [ ] OPTIONAL:  Hello @micheaol, this was a requirement in a previous milestone:

![image](https://user-images.githubusercontent.com/57421823/160487161-b995f52d-5b58-42fe-8d27-6726db591f01.png)

Therefore, I highly suggest you implement the author's input, as you will need it, and will make your life easier on the next milestones.  

## Explanation

The design that you can see in the sneak peek, is just for the styling part of the project.

## Hints

You could shorten the "Book title" input, and in that space insert your "Author" input, next to it, and you will preserve your styling.  Like this:

![image](https://user-images.githubusercontent.com/57421823/159839335-8935740f-33fe-4f87-9264-33a31dce9afb.png)

This is obviously optional (for now), but it will help you on the next milestones and make your life easier because you will be required to check the contents of that input to send the correct data to the API.

Also that way, you will be able to render your authors on the books list:

![image](https://user-images.githubusercontent.com/57421823/160488469-7465e1ec-7329-42db-88a7-43e2dbdbaae9.png)

I hope this is helpful to you! 💪

Reply:
Thanks you for your insightful comments. :+1: